spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -1885,10 +1885,8 @@ def foo(a)
   end
 
   context 'when converting a ruby class' do
-    it 'should yield \'PIntegerType\' for Integer, Fixnum, and Bignum' do
-      [Integer,Fixnum,Bignum].each do |c|
-        expect(calculator.type(c).class).to eq(PIntegerType)
-      end
+    it 'should yield \'PIntegerType\' for Integer' do
+      expect(calculator.type(Integer).class).to eq(PIntegerType)
```

Comment:
The original test needs to be retained for < Ruby 2.4 as we otherwise remove the assertion of how Fixnum/Bignum are mapped to an Integer PType. Make the test depend on Ruby version, and add new test for >= 2.4.